tests/main/listing/task.yaml

```diff
@@ -7,11 +7,14 @@ prepare: |
 execute: |
 
     echo "List prints core snap version"
+    # most core versions should be like "16-2", so [0-9]{2}-[0-9.]+
```

Comment:
I think this will be 16-2.24.1 (for example) rather than `16-2` which was a limitation of snapcraft and was never intentional.